### ISSUE TITLE
Missing numberPickerConfirmWidget parameter usage

### ISSUE THREAD

**Issue by Rafal-Okuniewski** [kaichii/pdf_viewer#12]
Please fix this code
```
  return PagePicker(
    title: widget.tooltip.pick,
    maxValue: widget.document.count,
    initialValue: _pageNumber,
  );
```
to
```
  return PagePicker(
    title: widget.tooltip.pick,
    maxValue: widget.document.count,
    initialValue: _pageNumber,
    numberPickerConfirmWidget: widget.numberPickerConfirmWidget,
  );
```
because parameter **numberPickerConfirmWidget** passing is missing.

**Comment by kaichii** [comment]
@Rafal-Okuniewski Fixed on 1.0.5.

**Comment by Rafal-Okuniewski** [comment]
Thank you very much.
